hubgrep/frontend_blueprint/templates/components/status_message/status_message.html

```diff
@@ -1,3 +1,3 @@
-{% macro message(status, msgStr, prefix="", affix="") %}
-    <div class="status-message status-{{ status }}">{{ prefix }}{{ msgStr }}{{ affix }}</div>
+{% macro message(status, content, prefix="", affix="") %}
+    <div class="status-message status-{{ status }}">{{ prefix }}{{ content }}{{ affix }}</div>
 {% endmacro %}
```

Comment:
rephrased argument since its possible to also pass and use any value, including markup